test/test_integration_pumactl.rb

```diff
@@ -136,6 +136,39 @@ def test_prune_bundler_with_multiple_workers
     wait_server
   end
 
+  def test_refork_cluster_mold_worker
```

Comment:
literally just copied fork_worker test and switched it to mold_worker, this is not adequate CI coverage by any means